woocommerce/woocommerce-android
# Store creation: Add tracking for Skip button clicks and finalize feature release

Part of: #8824 


### Description


This adds tracking for the Skip button usage on the various profiler screens:



| Event | Key | Values |
|--------|--------|--------|
| `site_creation_profiler_question_skipped` | `step` | `store_profiler_industries` or `store_profiler_commerce_journey` or `store_profiler_ecommerce_platforms` |

Additionally, as this is the last step in enabling store profiler, this PR also:
1. updates the release notes,
2. updates the feature flag to make it enabled to all users.

### Testing instructions

1. Create a new site,
3. On the industry profiler screen, tap "Skip". Ensure this tracking is shown on logcat: `🔵 Tracked: site_creation_profiler_question_skipped, Properties: {"step":"store_profiler_industries","is_debug":true}`
4. On the journey profiler screen, tap "Skip". Ensure this tracking is shown on logcat: `🔵 Tracked: site_creation_profiler_question_skipped, Properties: {"step":"store_profiler_commerce_journey","is_debug":true}`
5. Go back to journey profiler screen, and select "I'm already selling online",
6. On the platform profiler screen, tap "Skip". Ensure this tracking is shown on logcat: `🔵 Tracked: site_creation_profiler_question_skipped, Properties: {"step":"store_profiler_ecommerce_platforms","is_debug":true}`
